gateway/src/main/java/com/mx/path/gateway/connect/filters/HttpClientConnectException.java

```diff
@@ -0,0 +1,12 @@
+package com.mx.path.gateway.connect.filters;
+
+import com.mx.common.accessors.PathResponseStatus;
+import com.mx.common.connect.ConnectException;
+
+public class HttpClientConnectException extends ConnectException {
+  public HttpClientConnectException(String message, Throwable cause) {
+    super(message, cause);
+    withStatus(PathResponseStatus.UPSTREAM_SERVICE_UNAVAILABLE);
+    withReport(false);
```

Comment:
Do we want to report SSL errors to HoneyBadger?

Reply:
I believe these are getting reported but in a different spot. Currently we are missing details in the logs like this was a SSL error

Reply:
I'm not sure what that would tell us that a 531 in logstash wouldn't. ?

Reply:
I changed it to report. I don't think we will continue to do that, long term. But we need to put this fire out, so this will be helpful, for now.

Reply:
@RyanJones08, the facilities update (https://github.com/mxenabled/path-facilities/pull/43) should fix the issue with missing information in the logstash payload.